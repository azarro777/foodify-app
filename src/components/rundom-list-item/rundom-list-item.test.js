import React from 'react';
import RundomListItem from './rundom-list-item';
import { shallow } from 'enzyme';

describe('all props', () => {
  const props = {
    menuItem: [],
    refresh: function () {},
    addToFav: function () {}
  };

  describe('Testing <RundomListItem/>', () => {
    it('RundomListItem have rendered correctly', () => {
      const rundomListItem = shallow(<RundomListItem {...props} />);
      expect(rundomListItem).toMatchSnapshot();
    });
  });
});
