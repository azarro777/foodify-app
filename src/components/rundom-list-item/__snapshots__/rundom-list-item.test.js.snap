// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`all props Testing <RundomListItem/> RundomListItem have rendered correctly 1`] = `ShallowWrapper {}`;
